osu.Server.Queues.ScoreStatisticsProcessor/Helpers/S3.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using Amazon;
+using Amazon.Runtime;
+using Amazon.S3;
+
+namespace osu.Server.Queues.ScoreStatisticsProcessor.Helpers
+{
+    public static class S3
+    {
+        public const string REPLAYS_BUCKET = "score-replays";

```

Comment:
`osu-server-spectator` has this in an envvar. I considered doing the same but given that it was hardcoded in its only usage so far I figured maybe not. Open to changing.